ostd/libs/ostd-macros/src/lib.rs

```diff
@@ -217,3 +217,59 @@ pub fn ktest(_attr: TokenStream, item: TokenStream) -> TokenStream {
 
     TokenStream::from(output)
 }
+
+/// This macro is used to mark the page allocator initialization function.
+///
+/// # Example
+///
+/// ```ignore
+/// #[ostd::page_allocator_init]
+/// pub fn init_page_allocator() {
+///
+/// }
+/// ```
```

Comment:
Return anything, even an empty implementation in this function. The users can't tell what to do by reading it.